integration/helpers.go

```diff
@@ -124,6 +124,29 @@ func AssertEqualError(t testing.TB, expected mongo.CommandError, actual error) b
 	return assert.Equal(t, expected, a)
 }
 
+// AssertEqualErrorMessage asserts that expected error is the same as actual, ignoring the Raw part.
+func AssertEqualErrorMessage(t testing.TB, expected mongo.CommandError, altMessage string, actual error) bool {
+	t.Helper()
+
+	a, ok := actual.(mongo.CommandError)
+	if !ok {
+		return assert.Equal(t, expected, actual)
```

Comment:
here, it will fail always, isn't it? 
expected is type of `mongo.CommandError` and it's compared to actual which has another type in this if branch.

Reply:
Yes, it should fail here.

Reply:
In pseudocode:
```go
if !assert.ObjectsAreEqual(expected, a) { // compare with expected
  // compare with expected with altMessage
  expected.Message = altMessage
  return assert.Equal(t, expected, a)
}
```

Reply:
This is covered by code from the lines 141-145, isn't it?

Reply:
We still need it to cast to mongo.CommandError

Reply:
this block can be moved to the 141-145 if entirely, WDYT?

Reply:
It is also required for case when `actual` is `nil`